.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
```

Comment:
We can only add: - openjdk6
Sun jdk 6 is not available since it is end of life. 

Beyond that, simply put, I don't think we have the labor to be fixing openjdk6 only bugs.  So there is no value in having travis build that. We could add " - openjdk7" to the list as well. That could be more useful. I'd rather modify the list in a new PR though to be honest if we are only going to improve it.